### DISCUSSION TITLE
Repology down, causes Renovate to not update anything?

### DISCUSSION THREAD

**Discussion by bpfoster** [renovatebot/renovate#21730] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.48.2

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### If you're self-hosting Renovate, tell us what version of the platform you run.

7.21.7

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

I've got a repository that uses many data sources, for example docker, helm, and repology.

Looks like repology is down - website doesn't come up and renovate is reporting a timeout when attempting to query it.

It looks like renovate will not do any updates to any other items when repology is down - I see log message "External host error causing abort - skipping" and the expected updates to docker/helm files are not done.

Have I got that right - is this correct behavior?  Is there any easy way to disable the repology source while it is down so we can continue to receive other updates?

### What you tried so far.

Created testing repo.  Add in 1 helm file and it correctly updates a docker image in it.  Then add a repology-linked source and no updates are made.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by bpfoster** [comment]
I should have dug into the configuration more.  I see that I can disable repology with a packageRule
```
"packageRules": [
    {
      "datasources": ["repology"],
      "enabled": false
    }
```

I am still curious if you think this _should_ be the expected behavior.  Naively, I would have expected other updates to go through and just not see any that would have come in via Repology.

**Comment by rarkins** [comment, marked as answer]
It's expected behavior. Otherwise if you had a bunch of Repology PRs open then they'd be autoclosed
